json-endpoints.php

```diff
@@ -2189,146 +2189,6 @@
 	'group'       => '__do_not_document',
 	'stat'        => 'invites:1',
 	'method'      => 'POST',
-	'path'        => '/sites/%s/invites/%s',
```

Comment:
`invites/%s/delete`: [#1](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2165), [#2](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2305)

`invites/%s`: [#1](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2187), [#2](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2327)

`/sites/%s/users/%d`: [#1](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2126), [#2](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2266)

`/sites/%s/users/login:%s`: [#1](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2097), [#2](https://github.com/Automattic/jetpack/blob/master/json-endpoints.php#L2237) (difference here is the `group`.  'group' => 'users',` is correct one.)